### PR TITLE
module1:week 4 - Mobile-menu

### PR DESCRIPTION
**Pull request by uno36**
1. created a new branch
2. created a file script.js
3. made javascript code to toggle class list

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -15,6 +15,47 @@ body {
...
+  text-align: left;
+  padding: 12px 16px;
+  color: #fff;
+  text-decoration: none;
+  border-bottom: 1px solid #6f6c6b;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 700;
+}
```
Comment (Ibizugbe) [inline styles.css]: Nice job on implementing the functionalities of the mobile menu👍😉 But we have just a little bit of improvement relating to sticking to [the design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=nSrU7uvc0RxW1agh-0) as this is a requirement for [this milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md)😉. Kindly take note of the comparison given below😉

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|yours|design menu|

kindly take note of the font properties (color) and background color😉 and make the necessary change👍

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @uno36 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] On desktop size the hamburger menu button is not visible and it becomes visible on mobile👍.
- [x]  [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript), [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) has been correctly setup and they are passing👍.
- [x] When the user clicks (or taps) the hamburger button, the mobile menu appears and when the user clicks or taps the `x` button, the mobile menu disappears👍.
- [x] When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed👍.
- [x] When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears👍.
- [x] Nice animation and UI👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] Nice work on your commit messages as they are descriptive😉 but we can improve them by ensuring they are always capitalised😉 you can take a look at[ this lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more clarity😉

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @uno36      😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- Descriptive PR ✔️





## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by souad988 — APPROVED** [review]
Hi @uno36 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔Good design
✔Good documentation(PR title , commit messages)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**